shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -111,6 +111,7 @@ @implementation FlutterViewController {
   // UIScrollView with height zero and a content offset so we can get those events. See also:
   // https://github.com/flutter/flutter/issues/35050
   fml::scoped_nsobject<UIScrollView> _scrollView;
+  fml::scoped_nsobject<VSyncClient> _keyboardAnimationVsyncClient;
```

Comment:
nits: prefer to use property for Objective-C objects.

Reply:
Done